AI identities rework

Cre8or opened the pull request:
Fixes https://github.com/Bubbus/F3_CA_BUB/issues/51.

- Reworked AI identities to be applied locally, rather than via `remoteExec` (reduces network traffic)
- Added a function that (re)applies all custom AI identities on existing (spawned) units when the client initialises; this can either happen on mission init or JIP
- Implemented the `ENABLE_IDENTITY_REPLACEMENT` config macro, allowing mission makers to turn the component on or off

Cre8or committed:
AI identities rework

- Reworked AI identities to be applied locally, rather than via remoteExec (reduces network traffic)
- Added a function that (re)applies all custom AI identities on existing (spawned) units when the client initialises; this can either happen on mission init or JIP
- Implemented the ENABLE_IDENTITY_REPLACEMENT config macro, allowing mission makers to turn the component on or off

Cre8or committed:
Reverted identityAssignment.sqf

Rolling back to original values, as I modified them during testing.

Joecuronium reviewed (approved):
Teaches me to review.

Bubbus reviewed (commented):
good shit right here 👀

Cre8or committed:
Include dead units

- Modified fn_applyAllUnitIdentities to include dead units (may come in useful in some mission scenarios)
- Renamed fn_applyAllUnitIdentities to fn_applyExistingUnitIdentities to be more descriptive of its purpose

Cre8or committed:
Removed debug lines

derp

Bubbus reviewed (commented):
looks good, thank u ❤

i'll test this later on and merge it when i have confirmed the ingame behaviour.
